slack-notifier.go

```diff
@@ -46,9 +47,10 @@ func (s *Slack) Notify(check AppCheck) {
 		[]slack.Attachment{attachment})
 
 	webhooks := strings.Split(GetString(check.Labels, "alerts.slack.webhook", s.Webhook), ",")
+	proxy := strings.Split(GetString(check.Labels, "alerts.slack.proxy", s.Webhook), ",")
```

Comment:
1. Here `proxy` is an array right? but `slack.Send` takes in a string. You might want to fix that. 
2. Also the default value should be `s.Proxy` and not `s.Webhook`.